.circleci/config.yml

```diff
@@ -100,10 +100,12 @@ jobs:
 
       - install_java:
           version: adoptopenjdk-11-hotspot
-
+      - restore_cache:
+          keys:
+            - tarballs
       - run: sudo update-java-alternatives -s adoptopenjdk-11-hotspot-amd64 && java -version
-
-      - run: ./gradlew -i clean build --stacktrace
+      - run: mkdir ~/tarballs
```

Comment:
please can this `mkdir` be put inside gradle?

In hope that CI configurations are as thin as possible, so it is as simple as possible to repeat these steps outside of CircleCI, whether that be on locally or in https://ci-cassandra.apache.org/job/cassandra%7Esidecar/ 

(trivial here i know, just hoping that we'll see better practices and examples with a new project than what we have with that main project…)

Reply:
Good suggestion, I'll move it in.